packages/yourloops/components/dashboard-widgets/patient-info-widget.tsx

```diff
@@ -91,14 +92,17 @@ function PatientInfoWidget(props: PatientInfoWidgetProps): JSX.Element {
 
   const patientInfo: Record<string, string> = {
     patient: patient.profile.fullName,
+    gender: "",
     birthdate,
     email: patient.profile.email,
     hba1c: hbA1c ? `${hbA1c.value} (${hbA1c?.date})` : trNA,
   };
 
   const computePatientInformation = () => {
+    patientInfo["gender"] = genderLabels()[patient.profile.sex ?? ""];
```

Comment:
So, if the gender is not specified, the gender field is still displayed. Is that the wanted behaviour?

Reply:
Same question for remote monitoring invite dialog

Reply:
I kept the same behavior we have for the birthdate and hba1c that are also potentially empty field...